src/enclave/inside/EnclaveWasmModule.cpp

```diff
@@ -75,14 +75,12 @@ bool EnclaveWasmModule::callFunction(uint32_t argcIn, char** argvIn)
                                             argv.data());
 
     if (success) {
-        ocallLogDebug("Success calling WASM function");
+        SPDLOG_DEBUG_SGX("Success calling WASM function");
     } else {
         std::string errorMessage(
           ((AOTModuleInstance*)moduleInstance)->cur_exception);
-        // TODO - better logging
-        std::string errorText =
-          "Caught WASM runtime exception: " + errorMessage;
-        ocallLogError(errorText.c_str());
+        SPDLOG_ERROR_SGX("Caught WASM runtime exception: %s",
+                         errorMessage.c_str());
```

Comment:
I thought that spdlog formatting used `{}` not printf-style `%s`. Does this work?